std/conv.d

```diff
@@ -81,8 +81,16 @@ private auto convError(S, T)(S source, string fn = __FILE__, size_t ln = __LINE_
     if (source.empty)
         msg = "Unexpected end of input when converting from type " ~ S.stringof ~ " to type " ~ T.stringof;
     else
-        msg =  text("Unexpected '", source.front,
+    {
+        import std.range : ElementType;
+        ElementType!S el = source.front;
+
+        if (el == '\n')
+            msg = text("Unexpected '\\n' when converting from type " ~ S.stringof ~ " to type " ~ T.stringof);
+        else
+            msg =  text("Unexpected '", el,
```

Comment:
trying this leads to : "
std/format.d(1166): Error: function std.format.FormatSpec!char.FormatSpec.writeUpToNextSpec!(Appender!string).writeUpToNextSpec circular dependency. Functions cannot be interpreted while being compiled."